docs/openapi.yaml

```diff
@@ -120,6 +120,14 @@ paths:
                   schema:
                     type: integer
                     format: int32
+                - name: orderBy
+                  in: query
+                  schema:
+                    type: string
+                - name: orderAscDesc
```

Comment:
can we change it to `order `or `direction `mas?

Reply:
and please provide the example

Reply:
Noted mas, so instead of orderAscDesc, it will be direction ya?